inspire_dojson/hep/model.py

```diff
@@ -62,16 +62,17 @@ def convert_publication_infos(record, blob):
 
 def move_incomplete_publication_infos(record, blob):
     def _keys_with_truthy_values(d):
-        return [k for k, v in d.items() if v]
+        return set(k for k, v in d.items() if v)
 
     if not record.get('publication_info'):
         return record
 
     for publication_info in record['publication_info']:
-        if _keys_with_truthy_values(publication_info) == ['journal_title']:
+        if _keys_with_truthy_values(publication_info).issubset(set(['journal_record', 'journal_title'])):
```

Comment:
you can use literal set syntax

Reply:
Right, I also added a set comprehension above.